src/expr/src/relation/mod.rs

```diff
@@ -2115,43 +2116,88 @@ impl RowSetFinishing {
             && self.offset == 0
             && self.project.iter().copied().eq(0..arity)
     }
-    /// Applies finishing actions to a row set.
-    pub fn finish(&self, rows: &mut Vec<Row>) {
+    /// Determines the index of the (Row, count) pair, and the
+    /// index into the count within that paiir, corresponding to a particular offset
```

Comment:
For example, if offset is 42, and the row/count pairs are `(_, 10), (_, 17), (_, 20), (_, 11)`,

then this function will return `(2, 15)`, because after applying the offset, we will start at the row/diff pair in position 2, and skip 15 out of the 20 rows that represents.

Does it make sense with that example? If so, I will add it to the comment. If not, I will think of a better way to express this logic.

Reply:
Yep, makes sense! Would be great if you could include it in the doccomment! Thanks for the clarification.